efficientdet/keras/train_lib.py

```diff
@@ -430,14 +441,28 @@ class and box losses from all levels.
             box_loss_layer([num_positives_sum, box_targets_at_level],
                            box_outputs[level]))
 
-      if self.config.iou_loss_type and self.loss.get('box_iou_loss', None):
-        box_iou_loss_layer = self.loss['box_iou_loss']
-        box_iou_losses.append(
-            box_iou_loss_layer([num_positives_sum, box_targets_at_level],
-                               box_outputs[level]))
+    if self.config.iou_loss_type:
+      input_anchors = BoxList(
```

Comment:
Don't you want to wrap them as a function or a loss layer?

Reply:
Good, I do it later.